client/my-sites/plugins-wpcom/plugin-panel.jsx

```diff
@@ -22,8 +22,8 @@ export const PluginPanel = React.createClass( {
 				<Card className="wpcom-plugin-panel__panel-footer" href={ standardPluginsLink }>
 					{ this.translate( 'View all standard plugins' ) }
 				</Card>
-				<PremiumPluginsPanel />
-				<BusinessPluginsPanel />
+				<PremiumPluginsPanel siteSlug={ siteSlug } />
+				<BusinessPluginsPanel siteSlug={ siteSlug } />
```

Comment:
passing these `siteSlug` parameters adds in a new dependency that can be hard to mock. is there a reason we really need it? I would rather see us perform any site-specific logic in this file and hand down the information to the plugins panels that are more limited to what they want, such as the link for a given plugin or whether or not it's enabled

in fact, I don't see why we couldn't insert this information directly into the plugin data objects, even on the fly

```js
const linkInterpolator = siteSlug => plugin => {
	const { descriptionLink: link } = plugin;
	const descriptionLink = link.replace( '{siteSlug}', siteSlug );
	const isExternalLink = hasHostname( link );

	return { ...plugin, descriptionLink, isExternalLink };
}

const interpolateLink = linkInterpolator( siteSlug );
const businessPlugins = defaultBusinessPlugins.map( interpolateLinks );

<BusinessPluginsPanel plugins={ businessPlugins } />
```

maybe the naming could be better, but at least here we can still test in isolation and keep the concerns separate